swift/src/Ice/ServantManager.swift

```diff
@@ -12,7 +12,7 @@ final class ServantManager: Dispatcher {
     // when a servant is not found on a Swift Admin OA.
     private var adminId: Identity?
 
-    private var mutex = Mutex()
+    private var mutex = UncheckedMutex()
```

Comment:
We can't use Synchronization.Mutex to protect ServantManager's state because Dispatcher is not Sendable.